lib/tasks/evm_application.rb

```diff
@@ -102,11 +102,12 @@ def self.output_workers_status(servers)
            w.miq_server_id,
            w.queue_name || w.uri,
            w.started_on && w.started_on.iso8601,
-           w.last_heartbeat && w.last_heartbeat.iso8601]
+           w.last_heartbeat && w.last_heartbeat.iso8601,
+           (mem = (w.proportional_set_size || w.memory_usage)).nil? ? "" : mem / 1.megabyte]
```

Comment:
I would use `(mem = (w.proportional_set_size || w.memory_usage)).nil? ? "" : mem.to_s(:human_size)]`, but that's just splitting hairs

LGTM :+1: